### PR TITLE
updated to send unmarshalled record with schema to kafka topic

### PR DESCRIPTION
**Pull request by gurjit-sandhu**
updated to send unmarshalled record with schema to kafka topic and updated test cases

### CODE REVIEW

File: source/src/main/java/com/trustpilot/connector/dynamodb/utils/RecordConverter.java [hunk 1]
```diff
@@ -145,4 +148,65 @@ private String sanitiseAttributeName(final String attributeName) {
...
+                attributeValueStruct.put(attributeName, attributeValue.getNS());
+            } else if (attributeValue.getBS() != null) {
+                attributeValueStruct.put(attributeName, attributeValue.getBS());
+            } else if (attributeValue.getNULL() != null) {
+                attributeValueStruct.put(attributeName, attributeValue.getNULL());
+            } else if (attributeValue.getBOOL() != null) {
+                attributeValueStruct.put(attributeName, attributeValue.getBOOL());
+            }           
+        }
+        return attributeValueStruct;
```
Comment (mkravtsov-fetchrewards) [inline source/src/main/java/com/trustpilot/connector/dynamodb/utils/RecordConverter.java:179]: should probably have an else?  what if none of the else ifs match?
Also -> i'm sure you recognize the ugliness.  Any better way?

Comment (bgilbert-fetchrewards) [reply source/src/main/java/com/trustpilot/connector/dynamodb/utils/RecordConverter.java:179]: A switch maybe?

**Review by bgilbert-fetchrewards — APPROVED** [review]

File: source/src/main/java/com/trustpilot/connector/dynamodb/utils/RecordConverter.java (hunk 1 above)
Comment (gurjit-sandhu) [reply source/src/main/java/com/trustpilot/connector/dynamodb/utils/RecordConverter.java:179]: yep i will refactor some of this code once i make sure this is working with data in dev. will make it more cleaner and easy to read with comments

Comment (mkravtsov-fetchrewards) [reply source/src/main/java/com/trustpilot/connector/dynamodb/utils/RecordConverter.java:179]: I looked at the underlying AttributeValue.java.  hideous code.
private String n; -> screw you!  
to be clear, not gurjit's code.

**Pull request merged**